ztp/kube-compare-reference/optional/ptp-config/PtpConfigDualFollower.yaml

```diff
@@ -0,0 +1,132 @@
+# In this example, 2 ports of a single NIC are used in a active/standby configuration to synchronize to a 
+# remote grandmaster via interfaces $iface1 and $iface1
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: ordinary-clock-1
+  namespace: openshift-ptp
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  profile:
+  - name: oc1
+    phc2sysOpts: -a -r -n 24 -N 8 -R 16 -u 0
+    ptp4lConf: |-
+      [(?<iface1>[[:alnum:]]+)]
+      masterOnly 0
+      [(?<iface2>[[:alnum:]]+)]
+      masterOnly 0
+
+      [global]
+      #
+      # Default Data Set
+      #
+      slaveOnly 1
+      twoStepFlag 1
+      priority1 128
+      priority2 128
+      domainNumber 24
```

Comment:
Ditto here as above to add a capturegroup for the domainNumber.

Reply:
Fixed